Src/FluentAssertions/EventRaisingExtensions.cs

```diff
@@ -53,10 +55,12 @@ public static IEventRecording WithSender(this IEventRecording eventRecording, ob
         }
 
         /// <summary>
-        ///  Asserts that at least one occurence of the events had one or more arguments of the expected
-        ///  type <typeparamref name="T"/> which matched the given predicate.
-        ///  Returns only the events that matched both type and optionally a predicate.
+        /// Asserts that at least one occurrence of the events has some argument of the expected
+        /// type <typeparamref name="T"/> that matches the given predicate.
         /// </summary>
+        /// <returns>
+        /// Returns only the events having some argument matching both type and predicate.
+        /// </returns>
         public static IEventRecording WithArgs<T>(this IEventRecording eventRecording, Expression<Func<T, bool>> predicate)
         {
             Guard.ThrowIfArgumentIsNull(predicate, nameof(predicate));
```

Comment:
I see you are not documenting these kinds of exceptions (contract exceptions) today with `<exception` elements. Have you considered doing so to improve the API documentation?

I know this is not directly related to this PR here (which I found by accident by following a link from the latest release notes), but since it is related to documentation, I figured I'd mention here.

Reply:
Occurrences:
* `<exception`: 166
* `Guard.`: 224

So we mostly (~75%) do guard calls to `Guard.`, but sure, let's get that number closer to 100%.

Reply:
Ahhh I see, thought this was indicative of the trend but didn't really check it on my end, thanks for clarifying!